vpi/quickstart/helloworld/vpi.c

```diff
@@ -0,0 +1,51 @@
+#include <vpi_user.h>
+#include <stdio.h>
+
+/*
+ * Delay callback
+ */
+PLI_INT32 delay_ro_cb(p_cb_data data){
+    (void) data;
+    printf("Hello world! \n");
+
+    //vpi_control with vpiFinish asks the simulator to terminate
+    //the second argument is the return value
+    vpi_control(vpiFinish, 0);
+    return 0;
+}
+
+/*
+ * Main entry point
+ */
+void entry_point_cb() {
+
+    // VPI callback structure
+    s_cb_data cbData;
+    // VPI simulation time structure
+    s_vpi_time simuTime;
+    
+    // The callback is executed after a delay
+    cbData.reason = cbAfterDelay;
+    cbData.cb_rtn = delay_ro_cb;
+    cbData.user_data = 0;
+    cbData.value = 0;
+
+    // the callback is executed after a delay of 0
```

Comment:
From my ignorance, it seems that these lines (35-37) are optional. That is, the first example would also work with `cbData.time = NULL` (as per the common header above). However, I'm afraid that `cbData.time = NULL` and `cbData.reason = cbAfterDelay` might produce either a crash or weird behaviour.

Hence, I think that we should have three "hello" examples:

- `begin`/`hello`: same as `begin_end` but without `common_vpi.h`, or same as `helloworld` but without `simuTime`.
- `helloworld`
- `begin_end`

Anyway, we can move forward and I will take care of adding `begin`/`hello` after this PR is merged.

Reply:
Example `hello` is now included in #21.